hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
```

Comment:
Oh, I only mean in this string, not for all the scripts.  One sec, I'll make a quick patch so I can paste a diff to make it more clear what I'm suggesting.